src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# specify database access
+#TODO:
+# jpa properties are necessary for database update that's why we need them on prod
+# Store datasource credentials in variables?
```

Comment:
Klingt nach Environment Variables - vllt ein extra Ticket?

Reply:
Habe das bei anderer Gelegenheit auch schon so gesehen, dass man die Daten für den Datenbank-Zugang in ein eigenes env-File auslagert, das dann auch nur lokal exisitiert, d.h. also nicht hostet oder auf GitHub publiziert. Aber das sind Sicherheitsaspekte, die man auch später noch klären kann.